Pipfile

```diff
@@ -9,7 +9,7 @@ pylint = "*"
 
 [packages]
 python-telegram-bot = "*"
-psycopg2 = "*"
+psycopg2-binary = "*"
```

Comment:
change back ah, the pipfile.lock as well

Reply:
the mergeable push will change back